newIDE/app/src/SceneEditor/index.js

```diff
@@ -981,11 +982,11 @@ export default class SceneEditor extends React.Component<Props, State> {
   };
 
   zoomIn = () => {
-    if (this.editor) this.editor.zoomBy(0.1);
+    if (this.editor) this.editor.zoomBy(zoomInFactor);
```

Comment:
We could maybe round the zoom to the nearest `2 ** (i/16)` to avoid the steps to be shifted after a fit to content, but I guess it's not blocking.

Reply:
We'll see if the users notice this, in which case we could do this rounding indeed.